server/api/messageCenter/addMessage.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from 'express'
+import Requests from '@server/utils/requests'
+import { MessageCenterController } from '@server/controller/messageCenter'
+import { CountryIso } from '@core/country'
+import { AssessmentController } from '@server/controller/assessment'
+
+export const addMessage = async (req: Request, res: Response) => {
+  try {
+    const { countryIso, assessmentName, cycleName, key } = req.query
+    const { message } = req.body
+    const user = Requests.getRequestUser(req)
+
+    const { assessment, cycle } = await AssessmentController.getOneWithCycle({
+      name: String(assessmentName),
+      cycleName: String(cycleName),
```

Comment:
You can cast req.query to string record, so you don't have to cast each one separately;
`const { countryIso, assessmentName, cycleName, key } = <Record<string, string>>req.query`